src/Azure.Functions.ArtifactAssembler/ArtifactAssembler.cs

```diff
@@ -132,6 +146,14 @@ private static async Task<string> MoveArtifactsToStagingDirectoryAndExtractIfNee
         {
             await Task.Run(() => FileUtilities.CopyDirectory(artifactZipPath, destinationDirectory));
             await ExtractZipFilesInDirectoryAsync(artifactZipPath, destinationDirectory);
+
+            // Delete additional files that are not needed
+            var filesToBeDeleted = Directory.GetFiles(destinationDirectory);
```

Comment:
What are these additional files? 
Also, if the directory contains many files, it's best to use `Directory.EnumerateFiles` instead of `GetFiles` which loads all file names to memory.

Reply:
After extracting, we don't need the zip files anymore, the zip sha files, msi, etc

Reply:
also changed to EnumerateFiles as well